src/main/kotlin/net/atos/zac/identity/IdentityService.kt

```diff
@@ -60,4 +67,22 @@ class IdentityService @Inject constructor(
             .map { it.toUser() }
             .sortedBy { it.getFullName() }
     }
+
+    fun checkIfUserIsInGroup(userId: String?, groupId: String?) {
+        if (userId == null || groupId == null) {
```

Comment:
This does not make any sense in Kotlin I think. The idea is to make arguments like these non-nullable so that no code will call functions like this with nullable arguments, because why would you?

Reply:
The idea was to pass directly the rest objects that in the patch case might be missing user and group. This check will then be in `ZaakRestService`

Reply:
I think that is not how we should do it. Nullable function arguments like this do not make any sense in my opinion. I think the caller code should do these checks, not this function, which is simple to do using Kotlin.

Reply:
Check is now in ZaakRestService